azure/services/scalesets/scalesets.go

```diff
@@ -544,6 +544,7 @@ func (s *Service) getVirtualMachineScaleSetIfDone(ctx context.Context, future *i
 func (s *Service) generateExtensions() []compute.VirtualMachineScaleSetExtension {
 	extensions := make([]compute.VirtualMachineScaleSetExtension, len(s.Scope.VMSSExtensionSpecs()))
 	for i, extensionSpec := range s.Scope.VMSSExtensionSpecs() {
+		extensionSpec := extensionSpec
```

Comment:
This seems like `exportloopref` found a potential bug.

Reply:
how is this working right now? 🤔

Reply:
only one extension?

Reply:
yep that's probably it